ocs/agent/aggregator.py

```diff
@@ -257,12 +257,12 @@ def _check_for_duplicate_names(field_name, name_list):
         name_index = 1
 
         while field_name in name_list:
-            if len(field_name) < 252:
-                field_name += f'_{name_index:02}'
+            suffix = f'_{name_index:02}'
+            suf_len = len(suffix)
+            if len(field_name) < (255 - suf_len):
+                field_name += suffix
             else:
-                suffix = f'_{name_index:02}'
-                suf_len = len(suffix)
-                field_name = field_name[:-1*suf_len] + f'_{name_index:02}'
+                field_name = field_name[:-1*suf_len] + suffix
```

Comment:
This entire if block could be replaced by `field_name = field_name[:255-suf_len] + suffix`... but I'm being pedantic about an extreme edge case so it's probably safe to ignore me...